### PR TITLE
chore(other): CHECKOUT-8509 Remove experiment wrapping code for CHECKOUT-4183.checkout_google_address_autocomplete_uk

### PR DESCRIPTION
**Pull request by animesh1987**
## What?
Remove experiment wrapping for code to add autocomplete for UK addresses.

## Why?
We have rolled out google maps support for UK addresses for one year now, so seems like a good opportunity for removing the experiment code wrapping the change.

## Testing / Proof
- CI 

@bigcommerce/team-checkout

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-18.17
+20.6.0
```
Comment (TomA-R) [inline .nvmrc:1]: Seems like it should be a separate PR?

**Review by TomA-R — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (animesh1987) [reply .nvmrc:1]: Yes it can be, I am just taking the change from this https://github.com/bigcommerce/checkout-js/pull/1933 and applying it. Seems an easy and straightforward one.

Comment (TomA-R) [reply .nvmrc]: For anyone watching, we decided to move this to a different PR
